src/plugins/about/about.ejs

```diff
@@ -0,0 +1,69 @@
+
+<style>
+    paper-button {
+
+    }
+    paper-material{
+        margin-bottom:10px;
+        background-color:white;
+    }
+    .back-surface {
+        background-color:#f9f2f2;
+    }
+</style>
+<div class="back-surface">
+<paper-material  elevation="1">
+<fieldset>
+<legend>Support:</legend>    
+<paper-button raised id="dumpReport" on-tap="_dumpReport">Dump System Report</paper-button>
+</fieldset>
+</paper-material>
+<script>
+var template = $('#t')[0];
+var saveData = (function () {
+    var a = document.createElement("a");
+    document.body.appendChild(a);
+    a.style = "display: none";
+    return function (data, fileName) {
+        var json = JSON.stringify(data),
+            blob = new Blob([json], {type: "octet/stream"}),
+            url = window.URL.createObjectURL(blob);
+        a.href = url;
+        a.download = fileName;
+        a.click();
+        window.URL.revokeObjectURL(url);
+    };
+}());
+
+function formatDate(d) {
+
+  var dd = d.getDate()
+  if ( dd < 10 ) dd = '0' + dd
+
+  var mm = d.getMonth()+1
+  if ( mm < 10 ) mm = '0' + mm
+
+  var yy = d.getFullYear() % 100
+  if ( yy < 10 ) yy = '0' + yy
+
+  return ""+yy+mm+dd
+}
+
+
+template._dumpReport = function(){
+    template.cockpitEventEmitter.emit("plugin.about.dumpReport",function(report){
+        var result = JSON.parse(report);
+        result.browserString=navigator.userAgent;
+        result.timestamp=DateTime.now().toString();
```

Comment:
This is C#?. DateTime does not exist in javascript. Upon clicking DUMP SYSTEM REPORT, causes:

```
Uncaught ReferenceError: DateTime is not defined (program):37
```